parm/soca/obs/config/oc_aqua.yaml

```diff
@@ -3,7 +3,7 @@ obs space:
   obsdatain:
     engine:
       type: H5File
-      obsfile: !ENV ${DATA}/obs/${OPREFIX}oc_aqua.${PDY}${cyc}.nc4
+      obsfile: !ENV ${COM_OBS}/${OPREFIX}oc_aqua.${PDY}${cyc}.nc4
```

Comment:
Other point, I don't think we need `!ENV` anymore, R's parsing tool is smart now

Reply:
created #459 to tackle this @CoryMartin-NOAA .